trace_decoder/src/compact/compact_prestate_processing.rs

```diff
@@ -290,11 +317,11 @@ impl Header {
     }
 }
 
-#[derive(Debug)]
-pub(crate) struct WitnessOutput {
-    pub(crate) tries: PartialTriePreImages,
-    pub(crate) code: Option<HashMap<CodeHash, Vec<u8>>>,
-}
+// #[derive(Debug)]
```

Comment:
Is there a reason we're just commenting it out and leaving it?

Reply:
That should not be commented out, thanks for catching!